examples/vumps/vumps_2d_heisenberg.jl

```diff
@@ -34,9 +34,9 @@ function ITensorInfiniteMPS.unit_cell_terms(::Model"heisenberg2D"; width)
   opsum = OpSum()
   for i in 1:width
     # Vertical
-    opsum += -0.5, "S+", i, "S-", mod(i + 1, width)
-    opsum += -0.5, "S-", i, "S+", mod(i + 1, width)
-    opsum += "Sz", i, "Sz", mod(i + 1, width)
+    opsum += -0.5, "S+", i, "S-", mod(i, width) + 1
+    opsum += -0.5, "S-", i, "S+", mod(i, width) + 1
+    opsum += "Sz", i, "Sz", mod(i, width) + 1
```

Comment:
Since `i` goes up to `width`, doesn't that mean `mod(i, width)` doesn't do anything?